frontend/src/components/camp/PeriodSwitcher.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-toolbar-items v-if="period().camp().periods().items.length > 1">
+    <v-menu offset-y>
+      <template v-slot:activator="{ on, attrs, value }">
+        <v-btn
```

Comment:
Consider removing the left padding on this button so the text is properly left aligned with the contents of the ContentCard